dask/dataframe/tests/test_dataframe.py

```diff
@@ -4061,3 +4061,28 @@ def func():
 
     assert "['x']" in str(info.value)
     assert "['x', 'y']" in str(info.value)
+
+
+@pytest.mark.skipif(
+    PANDAS_VERSION < "0.25.0", reason="Explode not implemented in pandas < 0.25.0"
+)
+def test_dataframe_explode():
+    df = pd.DataFrame({"A": [[1, 2, 3], "foo", [3, 4]], "B": 1})
+    exploded_df = df.explode("A")
+    ddf = dd.from_pandas(df, npartitions=2)
+    exploded_ddf = ddf.explode("A").compute()
+    assert_eq(exploded_ddf.shape, exploded_df.shape)
```

Comment:
These three line can be replaced with:
> assert_eq(exploded_ddf, exploded_df) . 

assert_eq is comprehensive

Reply:
Can you also assert that ddf.divisions  == exploded_ddf.divisions ?

Reply:
Fixed